examples/vkcs_backup_plan/import.sh

```diff
@@ -0,0 +1 @@
+terraform import vkcs_backup_plan.mybackupplan 5dfe75cb-a00f-4bc8-9551-bd38f64747e7
```

Comment:
no reason to have import in examples since we can not check it
lets keep this in templates

Reply:
Moved to templates